src/gulpfile.js

```diff
@@ -255,6 +255,29 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor', 'add-revision-numbers'], function () {
+  var sources = gulp.src([
+    paths.build + '/template-editor/relayEditor.*.js'
+  ]);
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(sources, {
+    addRootSlash: false, // ensures proper relative paths
+    ignorePath: paths.build + '/template-editor' // ensures proper relative paths
```

Comment:
I do not understand your comments `// ensures proper relative paths`, could you give me some examples of how `addRootSlash` and `ignorePath` affect the _injection_?

Reply:
Of course, here you have some examples:

Suppose this full path: "http://.../example/myFile.js"

ignorePath: '/example' will filter the path with this result: /myFile.js

addRootSlash: will add (if not exist) or remove the first / for paths like "/myFile.js" with this results:
if it's true "myFile.js", and for false: "/myFile.js"

Reply:
Thanks your explanation was very useful, Now I can imagine how it works.